projectDocs/translating/readme.md

```diff
@@ -6,7 +6,8 @@ There are several tasks to be done, and this document hopes to give you an overv
 
 ## Translation Mailing List
 
-Translators should subscribe to the [NVDA translations mailing list](https://groups.io/g/nvda-translations) hosted at Groups.IO.
+To monitor changes to symbols and character description files, translators should subscribe to the [nvda-l10n](https://groups.google.com/a/nvaccess.org/g/nvda-l10n/about) mailing list.
+Translators should subscribe to the [NVDA translations mailing list](https://groups.io/g/nvda-translations) for general conversations regarding translations.
```

Comment:
I remember that with the API mailing list, I have not been able to subscribe with my usual e-mail address (`@laposte.net`); I have had to use an `@gmail.com` address. Or maybe I could subscribe but could not see the old messages.

Or do one need to create a Google account to be able to see the archived messages?

If my case is only an isolated case, you can ignore it. But if it's something more common, please double check that:
1. Users can subscribe with any e-mail
2. Users can access the messages archive, even with a non-gmail e-mail. If a Google account is required for this, maybe it's worth indicating it clearly.

Reply:
The message archive for both lists should be publicly available on the web. You can subscribe to them without a google account by email e.g. nvda-api+subscribe@nvaccess.org. The email to subscribe via is listed in both group descriptions

Reply:
OK. I have tested using in private browse mode and I confirm I can access the archives of the group without being identified.

Also I have subscribed to the API group with my @laposte.net e-mail, sending a subscribe message and replying to the join confirmation request message. And all seems OK: Ihave received the confirmation that I have been subscribed. The impossibility to subscribe in the past was very likely a mistake on my end. Sorry.